src/main/io/osd.c

```diff
@@ -153,6 +153,13 @@ static char osdGetAltitudeSymbol()
     }
 }
 
+static char osdGetBatterySymbol(int cellVoltage)
+{
+    /* Calculate current battery cell voltage as a percentage of range */
+    int voltPercent = ((cellVoltage - (batteryConfig()->vbatmincellvoltage * 10)) * 7) / ((batteryConfig()->vbatmaxcellvoltage - batteryConfig()->vbatmincellvoltage) * 10);
```

Comment:
Yes, that may be reasonable. But minVoltage is used for battery remaining percentage (https://github.com/betaflight/betaflight/blob/master/src/main/sensors/battery.c#L407).

Enabling blinking under vbatwarningcellvoltage is an option, but I don't know how blinking is implemented in OSD, it's probably not worth complicating things